### PR TITLE
[docs] various improvements

### PR DESCRIPTION
**Pull request by odow**
https://jump.dev/Convex.jl/previews/PR664/

### CODE REVIEW

File: test/test_utilities.jl [hunk 1]
```diff
@@ -171,6 +171,7 @@ function test_show()
     @test curvature(p) == Convex.ConvexVexity()
     @test sprint(show, p) == """
     Problem statistics
+      problem is DCP         : true
```
Comment (odow) [inline test/test_utilities.jl:174]: @ericphanson what do you think of this?

Comment (ericphanson) [reply test/test_utilities.jl:174]: Could be helpful? I don’t feel strongly either way

File: docs/src/manual/solvers.md [hunk 2]
```diff
@@ -1,84 +1,55 @@
...
-MILPs). Mosek and Gurobi can be used to solve SOCPs (even with binary or integer
-constraints), and Mosek can also solve SDPs. For up-to-date information about
-solver capabilities, please see the table [here](http://www.juliaopt.org/)
-describing which solvers can solve which kind of problems. See also
-[ConvexTests.jl](https://ericphanson.github.io/ConvexTests.jl/dev/) to see the
-results of running test problems with Convex.jl for many solvers.
-
-Installing these solvers is very simple. Just follow the instructions in
-the documentation for that solver.
+Install a solver usign the Julia package manager, replacing `"SCS"` by Julia
```
Comment (odow) [inline docs/src/manual/solvers.md:7]:
```suggestion
Install a solver using the Julia package manager, replacing `"SCS"` by Julia
```

File: docs/src/manual/solvers.md [hunk 3]
```diff
@@ -1,84 +1,55 @@
...
 As another example, if we wish to turn off printing for the SCS solver
 (that is, run in quiet mode), we can do so as follows:
-```julia
-using Convex, SCS
-const MOI = Convex.MOI
-
-opt = MOI.OptimizerWithAttributes(SCS.Optimizer, MOI.Silent() => false)
-solve!(p, opt)
+```@repl solvers
+silent_scs = MOI.OptimizerWithAttributes(SCS.Optimizer, MOI.Silent() => false)
```
Comment (odow) [inline docs/src/manual/solvers.md:44]:
```suggestion
silent_scs = MOI.OptimizerWithAttributes(SCS.Optimizer, MOI.Silent() => true)
```

**Pull request merged**